src/extension.ts

```diff
@@ -30,23 +28,23 @@ export async function activate(ctx: ExtensionContext) {
 }
 
 export async function intializePlugin(ctx: ExtensionContext) {
-  logIt(`Loaded ${getPluginName()} v${getVersion()}`);
+  console.log(`Loaded ${getPluginName()} v${getVersion()}`);
 
   //
   // add the player commands before we show the playlist
   //
   ctx.subscriptions.push(createCommands());
 
-  // init the music manager and cody config
-  const musicMgr: MusicManager = MusicManager.getInstance();
+  // migrate legacy spotify access token info to integration info
+  await migrateAccessInfo();
```

Comment:
Any potential unintended effects of doing the migration here vs. on the server?

Reply:
i don't think so. we have the integration like this in the backend already